iset.mm

```diff
@@ -73673,6 +73720,35 @@ product of extended reals (common case).  (Contributed by David A.
       ( cr wcel cmnf wne renemnf syl ) ABDEBFGCBHI $.
   $}
 
+  $( Plus infinity belongs to the set of extended reals.  (Contributed by NM,
+     13-Oct-2005.)  (Proof shortened by Anthony Hart, 29-Aug-2011.) $)
+  pnfxr $p |- +oo e. RR* $=
+    ( cpnf cr cmnf cpr cun cxr ssun2 cc cuni cpw df-pnf cnex uniex pwex eqeltri
+    cvv prid1 sselii df-xr eleqtrri ) ABACDZEZFUAUBAUABGACAHIZJPKUCHLMNOQRST $.
+
+  $( Plus infinity exists (common case).  (Contributed by David A. Wheeler,
+     8-Dec-2018.) $)
```

Comment:
What do you mean by "common case" ? That's the only case that I can think of...

Reply:
I believe that comment is intended to contrast it with http://us.metamath.org/mpeuni/pnfxr.html . I agree it isn't especially good wording but it is one we use a lot of places for convenience theorems like this.